pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -577,6 +577,11 @@ func init() {
 					Verbs:     sets.NewString("get", "create", "delete"),
 					Resources: sets.NewString("pods"),
 				},
+				// RecycleVolumeByWatchingPodUntilCompletion
+				{
+					Verbs:     sets.NewString("watch"),
```

Comment:
also, the linked PR looks like it adds event recording... does this not need write permission to events?